extension-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.1.0-rc.1",
+  "version": "1.4.0",
```

Comment:
last version in dev center is 2.2.0-rc.2

Reply:
continued github version